finder-app/test_files/a1.txt

```diff
@@ -0,0 +1,5 @@
+hello_world
```

Comment:
@MehulCUB consider whether you need to clutter your repo/diff by adding these files.  I'd only add them if you were using them to build new unit tests.